examples/cpp/visualize_openMVG_sfm_data/main.cpp

```diff
@@ -0,0 +1,96 @@
+#include <opencv2/opencv.hpp>
```

Comment:
Not required

Reply:
OpenMVG is already having an ImageIO that would work

Reply:
you are right I will use openMVG Image_Container and ImageIo instead

Reply:
OpenMVG is having image_io library too already.

Reply:
thanks @pmoulon for your tips, I will edit example and update the merge today